src/puppetlabs/puppetdb/pdb_routing.clj

```diff
@@ -52,7 +51,7 @@
                                              (format "%s/dashboard/index.html")
                                              rr/redirect))
                          (apply compojure/routes app-routes))
-      (mid/wrap-with-puppetdb-middleware #(:authorizer (get-shared-globals)))))
+      (mid/wrap-with-puppetdb-middleware (get-in defaulted-config [:puppetdb :certificate-whitelist]))))
```

Comment:
This one as well, and I think I noticed a few others, but perhaps not a big deal if they're not *too* long.